### DISCUSSION TITLE
Filter repositories by glob expression?

### DISCUSSION THREAD

**Discussion by magiconair** [lindell/multi-gitter#388] [Q&A]
We have thousands of repositories in our org and I'd like to use multi-gitter only for some of them which follow a certain naming pattern, e.g. `myorg/service-*`. Is that possible?

**Comment by lindell** [comment]
There is no way to do exactly that right now. But you can still archive this. 
1. Run on all repos in the org, and the first thing you do in your script is to match against the `REPOSITORY` env-var.
2. Set topics for those services, and use the topic filtering

Maybe #386 will help with this as well once it's implemented.

**Reply by magiconair** [reply]
Thank you. I thought just using a `glob` library would be sufficient but you'd have to get to the list of all repositories first.
